### ISSUE TITLE
G-Sync Compatible Issue

### ISSUE THREAD

**Issue by Dooofeee** [uberhalit/SekiroFpsUnlockAndMore#12]
My G-Sync compatible 144hz 48-144 range monitor disregards Nvidia control panel settings and stays locked at 60Hz even when the unlocker's running. I've changed all of the recommended settings in the Nvidia Control Panel prior to making this thread. 
Side note I noticed changing to boardless/window sets my monitor back to 144Hz. I'm going to try using Nvidia Inspector.
Edit: Deleting the Sekiro default profile using Nvidia Inspector and manually creating my own didn't resolve the problem.

Specs: R5 2600x, GTX 1050ti, Win10 Version 1803 KB4023057, 144hz 48-144 Acer KG240A

**Comment by Dooofeee** [comment]
![Sekiro](https://user-images.githubusercontent.com/49323756/55657427-307bb480-57c8-11e9-847b-c655580c6995.jpg)

Here's a picture with RTSS OSD on the left and my monitor's built-in refresh rate OSD on the right

**Comment by uberhalit** [comment]
Hi,
yes seems like the monitor locks down to default 60Hz in fullscreen. You can try disabling fullscreen "optimisations" for Sekiro by rightclicking the exe-> Compatibility-> tick Disable fullscreen optimizations.

![gg](https://winaero.com/blog/wp-content/uploads/2018/02/Windows-10-disable-fullscreen-optimizations-per-app.png)

Then you could try deleting the profile via **Nvidia Control Panel** and creating a new one with Profile Inspector. Starting the game in windowed mode and then switching to fullscreen once ingame has helped a few users too.

**Comment by Dooofeee** [comment]
No luck. I guess I'll turn off adaptive sync and lower quality settings instead.

 I wonder if AMD users are facing the same problem?

**Comment by uberhalit** [comment]
A few users have reported something similar, those that solved this had problems within their own system like corrupted drivers or misconfigurated windows settings. Make sure that Windows game bar and DVR is off and Game mode is turned on.

You can try to do hard-reset driver install:

1. Download [latest nvidia driver](https://www.nvidia.com/Download/index.aspx?lang=en-us)
2. Download [DDU](https://www.guru3d.com/files-get/display-driver-uninstaller-download,1.html)
3. Disconnect your internet so windows update won't auto-reinstall minimal drivers
4. Boot into safe mode
5. Uninstall GeForce Experience
6. Use DDU to completely erase the driver and evrything left from Nvidia
7. Reboot
8. Reinstall the previously downloaded driver
9. Reboot
10. Reactivate internet

**Comment by Dooofeee** [comment]
Quick update I found boardless Window and Windowed mode do work after a restart


![20190406_200801412_iOS](https://user-images.githubusercontent.com/49323756/55674737-b6f6cb80-5886-11e9-8b2e-b2ef1c654833.jpg)


![20190406_200824112_iOS](https://user-images.githubusercontent.com/49323756/55674738-b8c08f00-5886-11e9-9e47-38656e4dd239.jpg)

**Comment by uberhalit** [comment]
Did you have a look at the new troubleshooting guide?

**Comment by Forkinator** [comment]
Had a similar issue and wanted to share it here in case anyone else sees this (Uber, you already know this probably)

I could not get tearing to go away in Sekiro before I found out about this amazing tool. I had vsync on in control panel (latest drivers, always uninstall drivers with DDU, clean install of GPU drivers), and Gsync on in windowed + Fullscreen. I simply could not get the tearing to go away in fullscreen with Vsync on. 

As soon as i got this tool, I tried the frame rate unlock, and now I was getting above 60 fps, but still, tearing. 

Then I found out that Borderless windowed + Fullscreen stretch was the solution. When I enable Borderless Window, tearing goes away. It was beautiful. It was borderless fullscreen with 100 fps. Truly an amazing experience. I wonder what the developers were thinking keeping this game and previous Fromsoft titles locked to 60 fps if they run just fine at higher fps.


Huge thanks to Uberhalit. You truly made something great and you deserve recognition.
